app/javascript/dashboard/store/modules/bulkActions.js

```diff
@@ -0,0 +1,44 @@
+import types from '../mutation-types';
+import BulkActionsAPI from '../../api/bulkActions';
+
+export const state = {
+  uiFlags: {
+    isUpdating: false,
+  },
+};
+
+export const getters = {
+  getUIFlags(_state) {
+    return _state.uiFlags;
+  },
+};
+
+export const actions = {
+  process: async function createAutomation({ commit }, payload) {
```

Comment:
createAutomation?

Reply:
Added by mistake, was supposed to be processAction.

Fixing